### DISCUSSION TITLE
Svelte support?

### DISCUSSION THREAD

**Discussion by nickyhajal** [zed-industries/zed#6554] [Feature Requests]
Is there any way to get svelte support in Zed at the moment? Looking forward to being able to use it!

**Comment by JosephTLyons** [comment]
Hey all, could everyone who put an upvote-like emoji here go put a 👍 emoji on this issue? 

- https://github.com/zed-industries/community/issues/432

We track interest in issues using a script that counts the 👍 on issues.  The upvotes here, unfortunately, won't be counted.

**Comment by stolinski** [comment]
Svelte support is THE thing keeping me from giving this a better try.

**Comment by tylergannon** [comment]
I keep watching the blog for news of this feature but nary a word.  Myself and a lot of Svelte developers would gladly switch off of vscode and move to Zed full time.

Performance tweaks and git integration are exciting but are sadly useless to me while there's no support for the Svelte language server and syntax.

**Reply by JosephTLyons** [reply]
We have it scheduled to begin working on extensible language support this cycle (not sure how long it might take to add though).  This will unblock the community and enable them to build support for these things.

**Reply by tylergannon** [reply]
Super exciting.  FWIW I'm interested enough in text editor and stack choices that I've purposely bitten off a piece of my project that's pure TS, just so that I can be a beta user of Zed instead of nvim/vscode (oh and Bun instead of Node).  Looking forward to what you all come up with next!
